### PR TITLE
Consolidate Module menu in header navigation.

### PR DESCRIPTION
**Pull request by oliverguenther**
This effectively changes the following menus:

**Top Menu**
- Removed Administration item
- Added Item 'Work Packages' -> /work_packages
- Added Item 'News' -> /news
- Added Item 'Time Sheet' -> /time_entries (if user has a project with
      view time log permission)

**My Menu**
- Added Administration item as first position

With work packages, time entries, news now
residing in the top menu, these buttons are superfluous
in the projects overview and are also removed.

Relevant work package:
https://community.openproject.org/work_packages/20269
https://community.openproject.org/work_packages/20512

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://oliverguenther-feature-consolidate-module-menu.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://oliverguenther-feature-consolidate-module-menu.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: config/locales/en.yml [hunk 1]
```diff
@@ -964,6 +964,7 @@ en:
   label_this_week: "this week"
   label_this_year: "this year"
   label_time_entry_plural: "Spent time"
+  label_time_sheet_menu: "Time Sheet"
```
Comment (ulferts) [inline config/locales/en.yml]: "Sheet" needs to be lowercase

**Comment by ulferts** [comment]
Probably no good way around it, but merging this might break a few plugin cukes/specs.

**Comment by ulferts** [comment]
LGTM. 

Spelling needs to be fixed and travis has to agree.

**Comment by oliverguenther** [comment]
Did Hound just go berserk and used the default config for the spelling fix?

**Comment by Azure7111** [comment]
It did indeed. @houndci seems to be confused from time to time, though I don't know when or why.

**Pull request merged**